yarn.lock

```diff
@@ -2660,27 +2776,29 @@
     is-url "^1.2.2"
     json5 "^1.0.1"
     lru-cache "^6.0.0"
-    micromatch "^4.0.2"
+    micromatch "^4.0.4"
     node-forge "^0.10.0"
```

Comment:
It might be worth reaching out to the bundler team to see if they have plans to bump this dependency

Reply:
Just checked and there's a new bump! Will update the PR